bin/main.rb

```diff
@@ -0,0 +1,20 @@
+load '..\lib\player.rb'
+load '..\lib\game.rb'
+
```

Comment:
Lines 5-7 and 9-11 are essentially doing the same thing. It might not be much of a difference but it will make your code much cleaner if you make a method that will handle prompting users for their names and instantiating the corresponding player object.

Another thing I wish to ask here is that why did you need to make a main method when it's going to be immediately called in this same file? A cleaner alternative might be:

```
#This method is better inside another file and just invoke it here in main.rb:
def create_player(player_symbol)
   print 'Player who wish to choose #{player_symbol}:'
   name = gets.chomp
   Player.new(name, player_symbol)
end

#Instantiate objects:
player_1 = create_player("X")
player_2 = create_player("O")
board = Board.new(board_arguments here)
game = Game.new(player_1, player_2, board)

#Run game:
game.game_start
puts 'Do you want to player again? : (Y/N)'
wish = gets.chomp.to_s.upcase
main until wish == 'N'
```

Reply:
This is awesome modification. Really appreciate it.